parallel-consumer-python/pyallel_consumer/pom.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.confluent.parallelconsumer</groupId>
+    <artifactId>python</artifactId>
+    <version>0.1.0a0</version>
+    <properties>
+        <jackson.version>2.13.2</jackson.version>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>io.confluent.parallelconsumer</groupId>
+            <artifactId>parallel-consumer-core</artifactId>
+            <version>0.5.2.3</version>
+        </dependency>
+        <dependency>
+            <groupId>io.confluent</groupId>
+            <artifactId>kafka-avro-serializer</artifactId>
+            <version>5.3.0</version>
+        </dependency>
+        <dependency>
+            <groupId>io.confluent</groupId>
+            <artifactId>kafka-json-serializer</artifactId>
+            <version>7.2.1</version>
```

Comment:
remove - managed by maven rep management

```suggestion
```

Reply:
This dependency isn't specified in the rest of PC

Reply:
Ah, I thought we had some cp deps somewhere else, sharing a cp version number - turns out we don't. The only place we use cp version number is for the test container, inside the source. Ideally it'd be shared, but given this is outdated - all good.